api/nodes/serializers.py

```diff
@@ -1012,27 +1018,27 @@ def create(self, validated_data):
         return fork
 
 
-class ContributorIDField(IDField):
-    """ID field to use with the contributor resource. Contributor IDs have the form "<node-id>-<user-id>"."""
+class CompoundIDField(IDField):
```

Comment:
It didn't seem like this `ContributorIDField` was actually being used anymore, instead the `_id` was being stored in that format on the `Contributor` model. I decided to use this and generalize it anyway incase we ever wanted to use it in the future, but I'd also understand if we thought this was too broad and un-specific that it wouldn't be useful in other cases.